spec/spec_helper.rb

```diff
@@ -22,9 +22,8 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/all'
 require 'rspec/rails'
-# Add this to load Capybara integration:
-require 'capybara/rspec'
```

Comment:
This removes our integrations tests, which is not a good thing. Please add those lines back.